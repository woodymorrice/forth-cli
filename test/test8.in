true if false if ."NOPE" else ."YEP" then else ."NOPE" then
.
.
.