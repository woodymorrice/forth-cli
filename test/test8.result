1c1
< NOPE
---
> YEP
