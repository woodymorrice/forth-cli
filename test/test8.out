YEP
stack empty
stack empty
